content/influxdb/v2.0/upgrade/v1-to-v2/manual-upgrade.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Manually upgrade from 1.x to InfluxDB 2.0
+description: >
+  Manually upgrade from 1.x to InfluxDB 2.0
+menu:
+  influxdb_2_0:
+    parent: InfluxDB 1.x to 2.0
+    name: Manually upgrade
+weight: 11
+---
+
+To manually upgrade from InfluxDB 1.x to InfluxDB 2.0, the following tasks are required (at minimum):
+
+- [Migrate data](#migrate-data)
+- [Create authorizations](#create-authorization-token)
+- [Create DBRP mappings](#create-dbrp-mapping)
+
+{{% note %}}
+To automatically upgrade from InfluxDB 1.x to InfluxDB 2.0, use the [`influx upgrade` command](/influxdb/v2.0/upgrade/v1-to-v2/).
+{{% /note %}}
+
+## Create authorizations
+InfluxDB 2.0 requires authorization.
+If you do not have authorization enabled in 1.x, you will need to enable some kind of authorization
+either username and password, or [token](https://docs.influxdata.com/influxdb/cloud/reference/glossary/#token)-based authorization.
+
+Token-based authorization is the standard method in InfluxDB 2.0.
+We recommend using this method if possible.
+
+However, the [v1 compatibility API]() provides a wrapper for using username and password so that users can continue to use 1.x clients and libraries
+(that expect basic auth) with InfluxDB 2.0
+
+InfluxDB OSS 2.0 provides a 1.x compatible authentication API that lets you
+authenticate with a username and password like InfluxDB 1.x
+We call such authorizations _v1 compatibility authorizations_.
+(Compatibility authorizations are _separate from_ the credentials used to log into the InfluxDB user interface.)
+
+<!-- a v1 auth setup -- how to add a v1 auth username/password combo -->
+
+
+<!-- Note: influx v1 requires an authentication token -->
+### The `influx v1` command
+
+Use the `influx v1` command to TK.
+
+The `influx v1` is part of the InfluxDB 2.0 `influx` CLI.
+Like all operations in 2.0, `influx v1` commands must be authenticated with a token.
+
+
+When upgrading from 1.x to 2.0, this means you must 
+
+1. install 2.0
+2. create your username and password in the UI (remember, this is not he same as an authorization)
+3. in the UI, create a token
+
+Then, when completing the tasks below either pass this token on the command line, set as an environment variable, or setup a CLI config.
+
+#### Create a v1 compatibility authorization
+Use the [`influx v1 auth create` command](/influxdb/v2.0/reference/cli/influx/v1/auth/create/)
+to grant read/write permissions to specific buckets.
+Provide the following:
+
+- a list of [bucket IDs](/influxdb/v2.0/organizations/buckets/view-buckets/) for which to grant read or write permissions
+- a new username
+- a new password _(when prompted)_
+
+```sh
+influx v1 auth create \
+  --read-bucket 00xX00o0X001 \
+  --write-bucket 00xX00o0X001 \
+  --username example-user
+```
+
+#### View existing v1 authorizations
```

Comment:
Let's restructure headers (where it makes sense to do so)....for example, below "View DBRP mappings" is a subset of "Create DBRP mapping."